### ISSUE TITLE
Zcash Grant Proposal  - Video production, animation and marketing [Community Outreach]

### ISSUE THREAD

**Issue by publicimage** [ZcashFoundation/GrantProposals-2017Q4#23]
## **Motivation and overview**

Why are you doing this? What impact will the project have if it is completely successful?

Public Image Productions has a vested interest in cryptocurrencies. From the inception of blockchain technology, Public Image has sought to broaden general awareness and cultivate legitimacy of these exciting new markets with engaging high-end content.  Particularly, Zcash has long been an interest of Public Image Productions since we value privacy and security as the most crucial aspect to the legitimacy of cryptocurrency.  

Our primary objective is to enhance the breadth and scope of Zcash by reaching as many people as possible. 

Secondly, we aim to present Zcash's security-centric functionality and all associated technologies through clever, succinct animations. 

Through a series of sharp and precise videos we will illustrate the indispensable nature of zero-knowledge cryptography and help create awareness and education within the Zcash community and overall blockchain communities. We feel that short videos that can engage an audience thoroughly communicates intention, emotion and information in an inspiring way. 

## **Technical approach**

Through dynamic motion graphics and innovative animations, we will avail the general public of Zcash's innumerable advantages. We will work with the Zcash community to visualize and express the most up to date pressing issues, updates and areas with direct video content.  

## **Team background and qualifications**

Public Image Productions has produced high-end content for a wide range of clients; such as HBO, Delta and Deloitte. Expanding into the crypto-sphere we recently launched an introduction video for Santiment.net. 

We have also participated in many successful non profit initiatives by creating ad campaigns for Food For The Poor, NTSB and many others.  

Our team is composed of Adobe certified animators and editors utilizing an expansive range of software from Adobe After Effects to AutoDesk Maya.  

## **Evaluation plan**

Evaluation of a final product would best be figured out by a committee of administrators who can agree to a final version after various renditions. Public Image Productions always works hard and diligently to execute the proposed idea of the video and always makes sure to satisfy a project to completion. 

Here are some examples of final products: 

Public Image Reel:
https://vimeo.com/217017575/1164872c31

Deloitte's Upstream Tax Analyzer:
https://www.youtube.com/watch?v=wCzb9lW-ux0

Santiment.net:
https://vimeo.com/223019832    

## **Security considerations**

All videos will be property of Zcash. 

## **Schedule**

After the proposed idea for a video: 

We will begin by developing a detailed script and storyboard. Pending approval we will move to deliver a rough cut within 2 weeks. After a revision process, we will send a finalized version. Depending on length of video, animation/composition work and revision time, videos can be created and finalized rather quickly if need be.  An average time of completed projects in the examples/links above was 3 weeks from start to finish.  

## **Budget and justification**

Public Image Productions has a modular approach to budgeting. Our aim has always been to accommodate our clients needs within their existing financial framework. Our current budget includes script development, story-boarding, voice over recording, asset acquisition, motion graphics, 2D/3D animation, editing as well as time allocated for revisions. More time sensitive projects that require immediate attention may affect cost. The average cost of the examples provided are $3,500 - $5,000 USD, individually.  To be more cost efficient, Public Image Productions will create a package for multiple videos at a lower cost per video depending on the variables and attributes of the video content needed. 

We have also made videos quickly for events and fundraisers that don't require much time to put together and have a much lower cost. These videos of course retain high end graphic content consistent with Public Image Productions standard. We are a very flexible video production company.

**Comment by tromer** [comment]
1. To assess this, we need to come up with a tentative list of videos and brief descriptions of their content. *Community help needed!*

2. Is it feasible to open-source the videos, in an editable format (including the raw resources) that will let the community build up on this? And ensure that the resources you use carry a suitable license?
    (Ownership should not be transferred to Zcash Co., which is not a party to this Grant. Transferring ownership or assigning rights to the Zcash Foundation may be appropriate, but open-sourcing is best.)

**Comment by publicimage** [comment]
1. Precisely. We will be looking upon the Zcash forum and community to assess the most pertinent topics that will have the highest impact for Zcash and cryptocurrency. A poll or discussion with voted on topics may be a path to this insight. 

2. Transparency is key with the community. However, Public Image Productions creates animations and graphics from scratch. Keeping turnaround time in mind and in order to keep project files organized and compositions minimal, we feel any and all physical editing changes should come from us. Each animator has a specific style that pertains to their workflow which minimizes redundancy and maximizes efficiency -- Project files from various software tends to have a heavy workload and things can get quite messy if multiple editors are making changes. In a single composition up to 5 different software applications may be used to create a scene. The most effective way to utilize an open-source format would be to let the community view and make notes on various versions and drafts that Public Image Productions uploads all along each process. All resources that we use are royalty free or purchased per license which is included in each budget per video.

**Comment by tromer** [comment]
2. Makes sense. Can you suggest a concrete community-feedback process?

**Comment by publicimage** [comment]
Yes, I will research the best method suitable for community feedback. Preferably a method directly linked to Zcash - maybe via the forum or here on github.

**Comment by paulransfield** [comment]
An option for incorporating feedback on video content, that i use, is http://www.wipster.io - which allows comments to be made via a cloud platform, on the video footage and along the videos timeline.

**Comment by paulransfield** [comment]
Please consider increasing the scope of this proposal to include:
- instructional video material aligned to a universal digital literacy curriculum (say something from singapore)
- a learning management system ( I'd recommend Bridge from instructure )
- credential tokens ( i'd recommend credly - until zcash have their own )

**Comment by mineZcash** [comment]
First of all, I would like to say the videos given as examples are nicely done.

I think there are many subjects that could be addressed by quality video production.  Everything from comparing/contrast Zcash to Bitcoin/Ethereum/Other coins to more technical subjects like overviews of zkSNARKS would be great. Zcash Company has very few videos on thier official channel geared towards new users so this could definitely draw attention and adoption. 

I also like @paulransfield idea of more step-by-step instructional videos, but I wonder if that proposal should be addressed as it's own issue due to the apparent need for additional infrastructure that may be out of scope of what @publicimage had in mind.

**Comment by publicimage** [comment]
@mineZcash @tromer @paulransfield Thank you for all of your replies. 

Indeed there are many subjects video production could be useful in regards to the Zcash community. I'm glad the discussion has started. Over the next week I'll crawl the forum and maybe start a thread on what topics seem to yield the most attention and hopefully get some good feedback. I believe there is a need for general overview videos that Zcash could benefit from for newcomers as well as more specific videos for the inner community/back-end/hands on user. I'll report back here and I look forward to more comments and discussion. @mineZcash Thanks for the kind words.

**Comment by publicimage** [comment]
Checking in with a small update here. 

Periodically, I've been searching through the Zcash forum and I'm finding threads about the need for video content. Specifically a general video to start with, something that could be featured on the homepage of the Zcash site. The purpose of this video is to be direct with new impressions/newcomers to the Zcash site . This video should highlight Zcash's most pertinent key features. 

A good goal for runtime should be 90-120 seconds with an intelligent sounding voice for the voice over, paired with motion graphics and animations, helping to visualize the script. 

I worked on a logo tease in the meantime:
https://vimeo.com/235922761/a3f6464732

**Comment by acityinohio** [comment]
Every informal proposal has multiple reviews by the review committee. The reviews are being collected and discussed in a private google doc (the 5 reviewers all have edit access to it, no one else can view it). By way of early, informal feedback, the reviewers have made a list of projects that they consider leading candidates for grant funding.

In that vein, **your project was selected as one of the leading candidates**, and the review committee encourages you to submit a full proposal by **October 6th** and looks forward to reviewing it.

**Comment by acityinohio** [comment]
Also just a reminder @publicimage that the submission deadline is **October 6th**! Please endeavor to have a final proposal submitted by then, as an attachment to this issue (and yes, it can be October 6th anywhere in the world).

**Comment by publicimage** [comment]
@acityinohio We're working diligently as I type this message! We will have a full proposal submitted in time. Thanks for your consideration!

**Comment by publicimage** [comment]
[ZCASH_PROPOSAL.pdf](https://github.com/ZcashFoundation/GrantProposals-2017Q4/files/1364155/ZCASH_PROPOSAL.pdf)

**Comment by publicimage** [comment]
[ZCASH_PROPOSAL_Issue_23.pdf](https://github.com/ZcashFoundation/GrantProposals-2017Q4/files/1364158/ZCASH_PROPOSAL_Issue_23.pdf)

**Comment by ioptio** [comment]
Feeling really good about this proposal. Having an intro video for the website and an explainer video about the technology behind Zcash would be great for the seemingly impossible task of helping folks understand the "moon math". I do, however, wonder if we should keep trying to explain the moon math or if folks who are trying to understand it can learn with existing resources. Instead, perhaps a video showing the "life of a ZEC" and showing the various benefits to preserving privacy with different types of transactions.

**Comment by acityinohio** [comment]
@publicimage : I'm thrilled to inform you that the Grant Review committee—and the Zcash Foundation board—has tentatively approved your proposal! While the [recommendations are already posted](https://github.com/ZcashFoundation/ZcashFoundation/blob/master/grants/2017-Q4/Zcash-Foundation-grants-2017-Q4-recommendations.md), we are planning to make a more public post tomorrow morning (November 21st) Pacific Standard Time.

Next steps: please email me `josh [at] z.cash.foundation` with an email address suitable as a point of contact. Due to our [newfound 501(c)3 status](https://z.cash.foundation//blog/zcash-foundation-officially-nonprofit/) there are additional reporting and compliance burdens that may delay or change disbursements, but we are working through them as fast as we can.

Just in case you didn't see it, please find the committee recommendation for your project below, and congratulations again!

> The proposal is for producing two short videos, one "intro to Zcash" style and one more technical, including animation professional quality voice overs. The committee felt that videos are a good medium to engage Zcash users but were split as to what content needed to be covered. With guidance from the community, we feel that funding the production of the two videos that have been proposed would be beneficial to Zcash. Gibson Ashpool has also committed to assisting the producers to gather feedback from the community for the content of these videos. The proposer has shown impressive samples of the animation and marketing materials they have produced for other clients. Full funding of this grant is recommended.

**Comment by publicimage** [comment]
Excellent! We are extremely excited and honored for this amazing opportunity. Looking forward to starting this project!

**Comment by techoluwoye** [comment]
@publicimage 

Happy new Year hope all is well! My name is Oyedeji Oluwoye founder of Coincentrix Capital and fellow recipient of the Zcash Foundation Q4 grant. I am putting together material for the education of Zcash and would love to know if any progress or completed content has been made so if possible I could  include into my workshop material? Would like to connect and sync if possible regardless. 

Speak with you soon!

Regards,

Oyedeji Oluwoye

**Comment by publicimage** [comment]
Howdy Oyedeji!

Thanks for reaching out. Happy to see you are working on your grant project! We are too. 

You're more than welcome to read our tentative script and the responses we have received from the community. 

You can follow the thread here: https://forum.z.cash/t/zcash-introduction-video-zcash-foundation-awarded-grant/24502/12



Anything you think you could use, please do and keep me updated! 

All the best, 

Andrew

**Comment by sonyamann** [comment]
Hi @publicimage! I'm pinging all of the 2017 grant recipients for updates on their projects. Can you either write a paragraph or two here, or alternately email me? sonya@z.cash.foundation

Thank you!

**Comment by publicimage** [comment]
Hi @sonyamann ! Sure thing here is our update:

Since our approval of the 2017 grant proposal for the video production of (2) videos for the z cash foundation, the following are goals that have been met and goals that are ongoing. 

Proposed Video A: An introduction to ZCash / Explainer Video
Please follow our progress here: https://forum.z.cash/t/zcash-introduction-video-zcash-foundation-awarded-grant/24502/41

-We have established a presence in the zcash community forum and created a topic so the community could be involved in the decision making process, and as a guide for the production of the 1st video produced by Public Image Productions. The forum has offered a place for transparency between us (grant recipient) and the community. 

-Together, as a community, we have established a script. Initially written by Public Image Productions, then released to the forum for open discussion and criticism. Publicly revised, approved and finalized. 

-Voice over vote; Public Image Productions held a voice over talent audition, received over 50+ submissions, chose the top 10 and had the forum moderators conduct a community vote. After a deadline, a majority vote was in and Public Image Productions hired the winning voice over talent. 

-After working with the voice over talent, Public Image Productions released a complete 1st draft ZCash Introduction video and released it to the forum and community for open discussion. That video can be seen in the link provided above. 

-The introduction video was received with mostly favorable and satisfactory reviews, and a final version is underway. Just some timing & spacing fixes as well as a logo rework. 

Proposed Video 2 - A zk-SNARKS overview video

-Graphics have been built but not released. 

-A rough draft script has been written, also unreleased at this time but will be made public on the zcash forum in the upcoming week for open discussion. 

-Once the script is approved by the community, the voice over process will begin again and a video will be produced shortly after. 

That concludes our 6 month report! Let me know if you have any questions! Thank you!

**Comment by publicimage** [comment]
For ease of access, here is a private link (same that is featured in the zcash forum) to view Video 1, the Zcash explainer video: 

https://vimeo.com/272967601/156bc12408

**Comment by bars949** [comment]
> An option for incorporating feedback on video content, that i use, is http://www.wipster.io - which allows comments to be made via a cloud platform, on the video footage and along the videos timeline.

As for our basic needs in video production process we totally satisfied with [https://krock.io](https://krock.io/) and can definitely recommend this platform that gives the opportunity to control the projects at different stages, leave comments that are visible only to the team while making some adjustments, alert your team of a review need.
Meanwhile, the client monitors the progress and can make some amendments within the project cycle.
Adding a visual comment to the video file is the most useful option as it immediately becomes clear what needs to be fixed.
An interactive Gantt chart and calendar as a visual display of the team’s workload helps to understand that everything is under control.
